.github/workflows/individual-image-scanner-quay.yaml

```diff
@@ -0,0 +1,159 @@
+---
+name: Fetch security scan results from quay
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  scans:
+    runs-on: ubuntu-latest
+    outputs:
+      as-output: ${{ steps.access-setup-scan.outputs.VULNERABILITIES_EXIST }}
```

Comment:
Do not use abreviations. It will quickly become confusing.

Reply:
Fixed.